src/types/helpers.ts

```diff
@@ -59,9 +59,6 @@ export function validateTimedSequence(string: string): TimedSequence {
  * Return a random v4 UUID.
  */
 export function uuidv4(): UUID {
-  // Use uuidv4 from uuid package instead of crypto.randomUUID because randomUUID is not available in insecure contexts.
-  // This is safe for content scripts because they're in a separate JS context from the host page.
-  // https://developer.mozilla.org/en-US/docs/Web/API/crypto_property
   return v4() as UUID;
```

Comment:
We already have a lint rule for this. The comment is not really relevant anyway

Reply:
> We already have a lint rule for this. The comment is not really relevant anyway

Why isn't the comment relevant? It explains why we can't just use `crypto.randomUUID` (at one point we did try switching and it broke things). You could potentially move the explanation to the lint rule, but I am opposed to removing explanations of why we have to write code in a certain way.

Reply:
> Why isn't the comment relevant?

It takes about "safety" but it's confusing two concepts:

1. `crypto.randomUUID()` isn't made available to `http` because browsers want to push for HTTPS, it's not at all related to the "safety" of the API itself
2. content script or not the API is random and stateless

The only reason why we use `uuid` instead of `crypto.randomUUID()` is because we support `http:`, so I can restore this:

```suggestion
  // Use `uuid` package instead of `crypto.randomUUID()` the latter isn't available on HTTP-non-S pages
  return v4() as UUID;
```

Reply:
> The only reason why we use uuid instead of crypto.randomUUID() is because we support http:, so I can restore this:

OK, thanks for updating the context